inference-engine/src/mkldnn_plugin/mkldnn_exec_network.cpp

```diff
@@ -57,14 +57,14 @@ MKLDNNExecNetwork::MKLDNNExecNetwork(const InferenceEngine::CNNNetwork &network,
 
     if (cfg.exclusiveAsyncRequests) {
         // special case when all InferRequests are muxed into a single queue
-        _taskExecutor = InferenceEngine::ExecutorManager::getInstance()->getExecutor("CPU");
+        _taskExecutor = InferenceEngine::executorManager()->getExecutor("CPU");
```

Comment:
why don't we use `this->executorManager()` ? Maybe global one is already destroyed?

Reply:
fixed